packages/wallet-sdk/package.json

```diff
@@ -67,5 +64,6 @@
     "tsc-alias": "^1.8.8",
     "tslib": "^2.6.0",
     "typescript": "^5.1.6"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
hi hi, this line is added possibly because the yarn you used was coming from Node.js `corepack`, could you remove this line and try use a globally installed yarn to run yarn command and see if this line persists?

Reply:
yeah sure, I keep forgetting to remove it cuz it gets added every time I run yarn